src/plugins/discover/public/application/components/discover_legacy.tsx

```diff
@@ -193,29 +205,24 @@ export function DiscoverLegacy({
                 aria-label="Toggle sidebar"
```

Comment:
Shouldn't this `aria-expanded` be tied to `isSidebarClosed`?

Reply:
thx, finally done, misunderstood the initial comment :)